tests/parsing/FilterTest.py

```diff
@@ -126,3 +126,11 @@ def test_filter_by_can_detect_null(self):
             # All bear plus 1 line holding the closing colour escape sequence.
             self.assertEqual(len(stdout.strip().splitlines()),
                              TEST_BEARS_COUNT + 1)
+
+    def check_by_style(self):
```

Comment:
There's already a test for invalid filter `FilterTest.test_filter_by_unknown`. No need to another one. This test is failing. Update it accordingly to pass it.

Reply:
I think `py.test` will help you in checking your test changes

Reply:
@RaiVaibhav Its simply `pytest` now.